### PR TITLE
DOC:configuration.md: Change description of interval.

### PR DESCRIPTION
**Pull request by m-masataka**
Signed-off-by: Masataka Mizukoshi <m.mizukoshi.wakuwaku@gmail.com>
I've modified the ``docs/configuration.md``.   
The notifications ``interval`` and `` timeout`` was deprecated values.  
And Fixed some configuration related with timeout/interval.   
See this issue #2231

### CODE REVIEW

**Comment by m-masataka** [comment]
Sorry I accidentally closed this request.

File: docs/configuration.md [hunk 1]
```diff
@@ -816,9 +816,9 @@ notifications:
       disabled: false
       url: https://my.listener.com/event
       headers: <http.Header>
-      timeout: 500
+      timeout: 10s
       threshold: 5
-      backoff: 1000
+      backoff: 20s
```
Comment (stevvooe) [inline docs/configuration.md]: `20s` is probably too high. I would say `5-10s` would be better.

**Comment by stevvooe** [comment]
Not sure how this got this way. Here is the [development configuration](https://github.com/docker/distribution/blob/master/cmd/registry/config-dev.yml) for examples of correct values.

@dmcgowan PTAL

File: docs/configuration.md (hunk 1 above)
Comment (m-masataka) [inline docs/configuration.md]: Okey I see. Thank you for comment. I'll modify it.

**Comment by stevvooe** [comment]
LGTM

**Comment by dmcgowan** [comment]
LGTM

**Pull request merged**